pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -753,7 +753,7 @@ func (l *LibvirtDomainManager) generateConverterContext(vmi *v1.VirtualMachineIn
 		sev := kutil.IsSEVVMI(vmi)
 
 		if !l.efiEnvironment.Bootable(secureBoot, sev) {
-			log.Log.Reason(err).Errorf("EFI OVMF roms missing for booting in EFI mode with SecureBoot=%v, SEV=%v", secureBoot, sev)
+			log.Log.Errorf("EFI OVMF roms missing for booting in EFI mode with SecureBoot=%v, SEV=%v", secureBoot, sev)
```

Comment:
I share @maya-r view on https://github.com/kubevirt/kubevirt/pull/7567#issuecomment-1098107281 . I would prefer if you limited this PR to this single line. 

The comment changes you're adding are opinionated at best. 

The one about the logging is also superfluous.

Reply:
BTW - forgot to mention: good catch in this one.

Reply:
I've reverted most of the changes. I've left the ones that were fixed by my linter (adding space between `//` and the comment) - but if you want I can remove that as well.

Reply:
Out of curiosity, was it *your* linter (which) or the one used in the project ? 

Fell free to keep it, no problem - I won't insist.

Reply:
It's mine, `golangci-lint` to be exact which my IDE runs on save.